src/dist/nfs/nfs_client_impl.cpp

```diff
@@ -217,6 +233,8 @@ void nfs_client_impl::continue_copy()
             zauto_lock l(req->lock);
             const user_request_ptr &ureq = req->file_ctx->user_req;
             if (req->is_valid) {
+                _copy_token_bucket->consumeWithBorrowAndWait(req->size);
```

Comment:
Will `req->size` greatter than burst size?

Reply:
https://github.com/XiaoMi/rdsn/blob/704e1f0b00940cb3418583707154e19f9b2be0e6/src/dist/nfs/nfs_client_impl.cpp#L127 

As above, the requst size <= `nfs_copy_block_bytes`(nfs_copy_block_bytes = 4MB), so it has few influnce

Reply:
So I guess req_size is not definitely smaller than burst size. In this case, `consumeWithBorrowAndWait` will return false, So I suggest to add a judgment here.